src/smartcontracts/interaction.ts

```diff
@@ -182,6 +185,11 @@ export class Interaction {
         return this;
     }
 
+    withSender(sender: IAddress): Interaction {
```

Comment:
Is this necessary? It does not seem to be used in any test.

Reply:
This is used in `system-tests-js` repo.